packages/typescript-estree/tests/snapshots/typescript/expressions/tagged-template-expression-type-arguments.src.ts.shot

```diff
@@ -80,7 +80,7 @@ Object {
           "type": "Identifier",
         },
         "type": "TaggedTemplateExpression",
-        "typeParameters": Object {
+        "typeArguments": Object {
```

Comment:
I'm confused as to why the unit tests are failing on this rename. https://github.com/typescript-eslint/typescript-eslint/actions/runs/4116598038/jobs/7106955244

```plaintext
typescript/expressions/tagged-template-expression-type-arguments.src.ts
    expect(received).toEqual(expected) // deep equality
    - Expected  - 1
    + Received  + 1
    @@ -75,11 +75,11 @@
                  3,
                ],
                "type": "Identifier",
              },
              "type": "TaggedTemplateExpression",
    -         "typeArguments": Object {
    +         "typeParameters": Object {
                "loc": Object {
                  "end": Object {
                    "column": 8,
                    "line": 1,
                  },

      90 |
      91 |     // Received = Babel, Expected = TSESTree
    > 92 |     expect(babelAst).toEqual(tsestreeAst);
         |                      ^
      93 |   });
      94 | });
```

@bradzacher am I missing something?

Reply:
Because these old tests essentially enforce that the AST are the same, rather than cataloguing the differences like the new tests.

Which is why they suck and I wanna get rid of them.

You should add a pre-process step to the parse here
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/typescript-estree/tests/ast-alignment/utils.ts

Reply:
ok that does it I'm scripting it